listenbrainz_spark/recommendations/train_models.py

```diff
@@ -183,33 +292,31 @@ def main():
     current_app.logger.info('Training models...')
     t0 = time()
 
-    model, model_metadata, best_model_metadata = train(training_data, validation_data, num_validation, config.RANKS,
-        config.LAMBDAS, config.ITERATIONS)
+    model, model_metadata, best_model_metadata = train(training_data, validation_data, num_validation,
```

Comment:
I guess here best_model is actually quite accurate, no?

Reply:
sure